tool/tsh/tsh.go

```diff
@@ -517,7 +513,6 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	ssh.Flag("x11-trusted", "Requests trusted (insecure) X11 forwarding for this session. This can make your local displays vulnerable to attacks, use with caution").Short('Y').BoolVar(&cf.X11ForwardingTrusted)
 	ssh.Flag("x11-untrusted-timeout", "Sets a timeout for untrusted X11 forwarding, after which the client will reject any forwarding requests from the server").Default("10m").DurationVar((&cf.X11ForwardingTimeout))
 	ssh.Flag("participant-req", "Displays a verbose list of required participants in a moderated session.").BoolVar(&cf.displayParticipantRequirements)
-	ssh.Flag("request", "Attempt to request access to host if access is denied").BoolVar(&cf.doRequest)
```

Comment:
Shoudl we invert this and have a `--no-access-request` option if people want to opt out of the default behavior?

I'm asking here. "No" is a perfectly acceptable answer.

Reply:
I wonder if Kingpin automatically converts a user input of a flag starting with `--no` into false where the flag is Boolean with a default of true? `--disable` is fine though.

Reply:
yes, if I tried to add a flag `--access-request` then passing `--no-access-request` would work as false. But then the help string has to be for `--access-request`, and it's not even clear that the `--no` option exists from the help. But I could not get it to accept anything when trying to create the flag `--no-access-request` so I'm just going with `--disable`.